doc/guides/modules.rst

```diff
@@ -44,6 +44,9 @@ Module Repositories
 * The module repository codebase shall include a *module.yml* file in a
   :file:`zephyr/` folder at the root of the repository.
 
+* The *module.yml* file shall contain the ``name`` key and have the value
```

Comment:
*may

Reply:
please join here: https://github.com/zephyrproject-rtos/zephyr/pull/31156#discussion_r555135351

Reply:
Remember, those rules are for Zephyr community maintained Zephyr modules, not a rule for users creating downstream modules.
(yet one more reason why the modules guide should be split from the module contributions, hint: @ioannisg )